hr_hr.lang

```diff
@@ -1,215 +1,349 @@
-gui.xaero_entity_info_armour_numbers=Oklop u brojevima
-gui.xaero_entity_info_armour=Vrijednost oklopa
\ No newline at end of file
+gui.xaero_confirm=Potvrdi
+gui.xaero_cancel=Poništi
+gui.xaero_back=Natrag
+gui.xaero_delete=Obriši
+gui.xaero_OK=U redu
+gui.xaero_add_edit=Dodaj/Uredi
+gui.xaero_new_waypoint=Nova putna točka
+gui.xaero_initial=initial
+gui.xaero_edit_waypoint=Uredi putnu točku
+gui.xaero_armour_status_settings=Postavke stanja oklopa
+gui.xaero_choose_a_preset=Odaberi predložak
+gui.xaero_choose_an_interface=Odaberi sučelje
+gui.xaero_armour_status=Stanje oklopa
+gui.xaero_notifications=Obavijesti
+gui.xaero_minimap=Mini karta
+gui.xaero_miscellaneous=Razno
+gui.xaero_edit_mode=Način uređivanja
+gui.xaero_instructions=Upute/Pomoć
+gui.xaero_not_ingame=Moraš biti u igri za korištenje načina uređivanja!
+gui.xaero_welcome_edit_mode=Dobrodošli u način uređivanja!
+gui.xaero_howto_select=Lijevi klik na nekom od sučelja za označavanje.
+gui.xaero_howto_drag=Lijevi klik i povlačenje na odabranom sučelju za pomicanje tog sučelja.
+gui.xaero_howto_deselect=Desni klik za odznačavanje odabranog sučelja.
+gui.xaero_howto_center=Pritisni "C" kako bi centrirao i decentrirao sučelje
+gui.xaero_different_centered=Neka sučelja imaju drukčiji centrirani izgled.
+gui.xaero_howto_flip=Pritisni "F" da okreneš ili vratiš označeno sučelje.
+gui.xaero_howto_settings=Pritisni "S" za brzi pristup postavkama označenog sučelja.
+gui.xaero_howto_preset=Odaberi predložak klikom na gumb "Odaberi predložak".
+gui.xaero_howto_save=Spremi promjene klikom na gumb "Potvrdi".
+gui.xaero_howto_cancel=Poništi promjene klikom na gumb "Poništi".
+gui.xaero_minimap_settings=Postavke mini karte
+gui.xaero_miscellaneous_settings=Razne postavke
+gui.xaero_notification_settings=Postavke obavijesti
+gui.xaero_better_pvp_settings=Better PVP postavke
+gui.xaero_server_disabled=Poslužitelj je možda onemogućio neke mogućnosti moda.
+gui.xaero_waypoints=Putne točke
+gui.xaero_box_potion_effects=Sučelje koje prikazuje trenutno aktivne efekte napitaka.
+gui.xaero_box_armour_status=Sučelje koje prikazuje oklop koji trenutno nosiš i predmet u ruci.
+gui.xaero_box_sprint=Trčanje držanjem samo jedne tipke (podesivo).
+gui.xaero_box_sneak=Omogućuje uključivanje/isključivanje šuljanja i stajanje šuljajući se bez potrebe držanja tipke.
+gui.xaero_box_notifications=Obavijesti na zaslonu koje prikazuju različite stvari kao što su: niska glad, nisko zdravlje, eksplozija u blizini ili kad si pogođen strijelom/ama.
+gui.xaero_box_xp=Bodovi iskustva koje si dobio ili izgubio lete preko tvog zaslona.
+gui.xaero_box_quick_use=Sistem tipki sličan akcijskoj traci koji ti dopušta automatsko korištenje određenog predmeta u svojoj vrućoj traci bez potrebe za pomicanjem sa svojeg trenutno držanog predmeta. Pritiskanje određene tipke omogućuje ti da nastaviš s korištenjem predmeta kao da si pritiskao desni gumb na mišu. Npr.: postavljanje baklji, TNT-a, bacanje napitaka, jedenje hrane, pijenje.

```

Comment:
I thought at the time that "Pritiskanje" wold be fine, but I'll change it into "Držanje". Because "Držanje" is like holding a object, example a baby, and not a button on keyboard, It was more natural to have "Pritiskanje" as it means pushing down on something for a long time, example being a key on keyboard or piano. I get why you think that it could be confusing for people, but in game, that is how it is translated. Changing it here would add another word meaning the same and that would be bad. Anyways, I'll change it.

Reply:
You know better than me, so if you feel like Pritiskanje works better, then you should keep it. "Pushing down on something for a long time" is exactly what we need.